package.json

```diff
@@ -10,14 +10,14 @@
     "backend-issuer"
   ],
   "scripts": {
-    "lint":                 "npm run lint --workspaces",
+    "lint":                 "eslint ./test && npm run lint --workspaces",
```

Comment:
@msterle is this running lint in test files now? Do we usually do this?

Reply:
This is to lint our e2e tests in addition to running the npm `lint` script in each of the workspaces, however those packages define the `lint` command